README.md

```diff
@@ -1 +1,121 @@
+As a Lad you are only able to strike your starting nail. As a Lord you are allowed to strike the 3 center nails.
+The game ends when the 3 center nails have been hammered completely into the stump. The winner is the person who dealt the final strike to the last nail.
+
+
+### Taking a turn
+A turn consists of 3 phases:
+* Flip phase
+* Strike phase
+* Drink phase
+
+
+#### Flip
+During the flip phase, a player throws the hammer into the air so that it flips vertical (head over handle). 
+The player then attempts to catch them hammer after it flips at least once in the air.
+
+
+The number of vertical flips is the number of strikes the player may make in the Strike phase.
+
+
+The player may not adjust their grip on the hammer. 
+The way that player catches the hammer is how they must perform their strikes.
+
+
+Upon catching the hammer, the player must immediately move to the Strike phase without hesitation.
+
+
+If a player fails to catch the hammer, they must drink and immediately end their turn, passing the hammer to the next play in turn order.
+
+
+#### Strike
+Upon catching the hammer, the player must immediately perform a strike.
+The player may not hesitate or aim, though they are allowed to bring them hammer up and down again after catching.
+
+
+If that player is a Lad, they must strike their starting nail.
+If they are a Lord, they must strike one of the center nails.
+
+
+If a Lad hammers their starting nail completely into the stump, they instantly ascend to the rank of Lord. 
+This can happen between strikes.
+I.e. if a Lad has two strikes because he or she performed a double hammer flip, and they drive their starting nail completely into the stump on the first hit, the next hit must be performed on a center nail as Lord.
+
+
+A nail is considered completely hammered into the stump if all plays agree or if the Hammer Test decides it is (see Hammer Test). Any player may challenge a nail and perform a Hammer Test. This effectively pauses the game. Giving away the hammer for someone to perform the Hammer Test does not end your turn, and that player must return the hammer to you upon completing the Hammer Test, resuming the game.
+
+
+Each successful strike on a nail grants the player one drink to give out during the Drink phase. 
+Each miss is one drink the player must take themselves during the Drink phase.
+
+
+If a player hits the wrong nail it counts as a miss. If doing so results in the final center nail being hammered into the stump, A new center nail is created and the current player is removed from the game.
+
+
+#### Drink
+A "Drink" is a consistent unit which must be decided upon before the game begins.
+
+
+During the drink phase the player may give out drinks for each successful strike they made during the Strike phase. 
+For each strike made as a Lad, that player may give a drink out to another Lad. For each strike made as a Lord, that player may give a drink out to another Lord or make all remaining Lads drink at once.
```

Comment:
Maybe explicitly say here that a Lad can't ever make a Lord drink. It's implied, but might be good to be extra clear.